src/applications/medical-copays/routes.jsx

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
-import { Route, Redirect } from 'react-router-dom';
+import { Route } from 'react-router-dom';
 import MedicalCopaysApp from './containers/MedicalCopaysApp.jsx';
 import OverviewPage from './containers/OverviewPage';
 import DetailPage from './containers/DetailPage';
 
 const Routes = () => (
   <MedicalCopaysApp>
-    <Route exact path="/">
-      <Redirect to="/copays" />
-    </Route>
     <Route component={OverviewPage} path="/copays" />
     <Route component={DetailPage} path="/copay-detail" />
```

Comment:
probably need to add exact on these routes I had a bug with not adding it on routes on debt-letters. This seems to only be the case for react-router-dom rather than just plain react-router.

Reply:
cool ill add it tks